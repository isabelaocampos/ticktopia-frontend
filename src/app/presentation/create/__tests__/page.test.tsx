import { render, screen } from '@testing-library/react'
import CreatePresentationPage from '../page'


// Mock the PresentationCreateForm to isolate the test
jest.mock('../../../../features/presentation/components/PresentationCreateForm', () => () => (
  <form data-testid="presentation-create-form"></form>
))

describe('CreatePresentationPage', () => {
  it('renders the heading', () => {
    render(<CreatePresentationPage />)
    expect(screen.getByRole('heading', { name: /crear presentaciÃ³n/i })).toBeInTheDocument()
  })

  it('renders the PresentationCreateForm component', () => {
    render(<CreatePresentationPage />)
    expect(screen.getByTestId('presentation-create-form')).toBeInTheDocument()
  })

  
})
