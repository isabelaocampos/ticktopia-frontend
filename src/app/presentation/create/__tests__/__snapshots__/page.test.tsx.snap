// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreatePresentationPage matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="max-w-xl mx-auto mt-10 p-4 border rounded shadow space-y-6"
  >
    <h1
      class="text-2xl font-bold"
    >
      Crear Presentaci√≥n
    </h1>
    <form
      data-testid="presentation-create-form"
    />
  </div>
</DocumentFragment>
`;
